src/cpu/xbyak/xbyak_util.h

```diff
@@ -115,7 +117,7 @@ class Cpu {
 	}
 	void setFamily()
 	{
-		unsigned int data[4];
+		unsigned int data[4] = {0};
```

Comment:
Unrelated change.

Reply:
It is related. We use this to zero out all the features.